### PR TITLE
Enable pickup chat notification emails

### PR DESCRIPTION
**Pull request by tiltec**

### CODE REVIEW

File: foodsaving/conversations/emails.py [hunk 1]
```diff
@@ -22,17 +35,73 @@ def prepare_conversation_message_notification(user, message):
...
-            from_email=from_email,
-            user=user,
-            reply_to=[reply_to],
-            context={
-                'conversation_name': conversation_name,
-                'author': message.author,
-                'message_content': message.content_rendered(),
-                'conversation_url': group_wall_url(group),
-                'mute_url': conversation_mute_url(group, message.conversation)
+    return prepare_email(
```
Comment (nicksellen) [inline foodsaving/conversations/emails.py]: Why did you remove the translation override?

Comment (tiltec) [reply foodsaving/conversations/emails.py]: It seems unnecessary, as `prepare_email` has its own translation override.

Comment (nicksellen) [reply foodsaving/conversations/emails.py]: Ah, yes I see. Makes sense. And for the other one it does translation stuff in the preparation so it does need it.
